src/NuGet/EntityFramework/tools/EntityFramework6.psm1

```diff
@@ -47,7 +47,7 @@ function Add-EFProvider
         [parameter(Position = 2, Mandatory = $true)]
         [string] $TypeName)
 
-    $configPath = GetConfigPath($Project)
+    $configPath = GetConfigPath $Project
```

Comment:
These weren't wrong, just confusing. A second argument would have to be added after the closing parameter `F($x) $y $z`